FogCreek/Glitch-Community
# Pebble-fragrance: only show 'skip collections section' buttons when there are collections to skip

## Links
* pebble-fragrance.glitch.me
* https://github.com/FogCreek/Glitch-Community-Work/issues/331

## GIF/Screenshots:
https://media.giphy.com/media/iI9h1CNvDgTcjwQKJa/giphy.gif

https://media.giphy.com/media/U7Q2LBqBpWe2V9XorE/giphy.gif

## Changes:
* Only show the 'skip collection section' buttons if there are some collections to skip over

## How To Test:
* tab through a profile page with collections; see the buttons
* tab through a profile page of your own logged-in user that has no collections, see that there are no skip buttons

## Feedback I'm looking for:
Anything
